packages/zoe/src/contracts/sellItems.js

```diff
@@ -90,7 +91,7 @@ const start = zcf => {
     // Check that the wanted items are still for sale.
     if (!AmountMath.isGTE(currentItemsForSale, wantedItems)) {
       const rejectMsg = `Some of the wanted items were not available for sale`;
-      throw buyerSeat.fail(new Error(rejectMsg));
+      throw buyerSeat.fail(assert.fail(rejectMsg));
```

Comment:
`assert.fail` throws. It's not a replacement for `new Error`.

I suggest just taking out `new`. `Error(...)` and `new Error(...)` mean the same thing.

Reply:
Better would be `assert.error`. Like `Error(...)` it makes and returns an error. Like the other `assert` methods, you can use an
```js
X`literal stuff ${redacted value}`
```
for better diagnostics. Also, all the `assert` methods create the error object at the same line of code that becomes a useful place to put a breakpoint.